examples/cp-all-in-one/docker-compose.yml

```diff
@@ -18,17 +18,17 @@ services:
     depends_on:
       - zookeeper
     ports:
-      - "9092:9092"
       - "29092:29092"
```

Comment:
Nit: this doesn't need exposing if it's only being used within the Docker network. It's only 9092 that needs exposing.